### ISSUE TITLE
MO2 fails to install (after commit that locks MO2 version to 2.4.4)

### ISSUE THREAD

**Issue by danaYatsuta** [sonic2kk/steamtinkerlaunch#990]
## System Information
- SteamTinkerLaunch version: git
- Distribution: Arch Linux
- Installation Method: from AUR with yay

## Issue Description
Running `steamtinkerlaunch mo2 start` after updating to the latest version fails, even after removing .config/steamtinkerlaunch and /dev/shm/steamtinkerlaunch directories. Attempted with both Proton GE from AUR and Proton 8.0.4. It looks like MO2 fails to install altogether, as there is no compatdata/pfx/drive_c/Modding directory, where MO2 is usually installed to. This seems to be an issue with downloaded installer - downloads/mo2/Mod.Organizer-2.4.4.exe, as it takes up 0 bytes.

## Logs

Console output:
```
$ steamtinkerlaunch mo2 start
grep: invalid option -- '\'
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
```
[steamtinkerlaunch.log](https://pastebin.com/raw/Tis3URsc)

Installer's size:

```
$ du -h Mod.Organizer-2.4.4.exe 
0	Mod.Organizer-2.4.4.exe
```

**Comment by sonic2kk** [comment]
Seems I commented out a bit that I shouldn't have, will push a fix now.

**Comment by sonic2kk** [comment]
Fixed with 9b6b94c9a3d90bc74b3e5d5b833c3f01d738262f, the grep errors were a good help too. Verified that MO2 downloads now and is not empty :-) Thanks for catching this.
